code/style.css

```diff
@@ -231,3 +232,14 @@ button {
     border: none;
   }
 }
+
+
+.counter {
+  display: flex; 
+  border: none;  
+  margin-top: 20px; 
+  background-color: #36d1c4;
```

Comment:
the color of the guesses counter could maybe have been another color, it is slightly hard to read the text in it :)